privaterelay/locales/en-US/app.ftl

```diff
@@ -231,11 +231,8 @@ onboarding-tip-3 = Via the context menu, just right-click (Windows) or Control-c
 modal-rename-alias-saved = Label saved!
 modal-delete-headline = Permanently delete this alias?
 
-# Variables:
-#   $attrs (string) - specific attributes added to HTML element
-#   $email (string) - User email address
 modal-delete-warning-recovery-html = Once you delete this alias, it cannot be recovered. 
-    { -brand-name-firefox-relay } will no longer forward messages sent to <span { $attrs }>{ $email }</span>, including messages that allow you to reset lost passwords.
+    { -brand-name-firefox-relay } will no longer forward messages sent to <strong></strong> { $email }, including messages that allow you to reset lost passwords.
```

Comment:
I recommend removing the `$email` variable as it is no longer used. 

```suggestion
    { -brand-name-firefox-relay } will no longer forward messages sent to <strong></strong>, including messages that allow you to reset lost passwords.
```

I also recommend keeping the `{ $attrs }` variable (and the lines #234/235 that were removed). This was we can add a specific `js-` class name to the `strong` attribute and target it more directly in the JavaScript.

Reply:
Disregard the second recommendation. Go ahead and remove both `$email` AND `$attrs` variables.